daemon/event.go

```diff
@@ -65,19 +69,31 @@ func (d *daemon) CreateEvent(req *pb.CreateEventRequest, resp pb.Daemon_CreateEv
 
 	finishTime, _ := time.Parse("2006-01-02", req.FinishTime)
 
+	// handling booked events might be changed
+	startTime, _ := time.Parse(time.RFC3339, req.StartTime)
```

Comment:
why do we have so many different time formats? (time.RFC3339, "2006-01-02", displayTimeFormat). can we use only one or the entire repo?

Reply:
Hmm, for booking events it is a little bit old fashion, I mean, like when you do not care hour and minutes then, the likelihood of having `start event` order at same time is very high which might have negative effect on server side. Like imagine a scenario where three different event booked for  date `2020-07-10`, actually it means following ; 

`2020-07-10 00:00:00`  - for all events which are booked for same date. 

Then event scheduler will check events and see all three, will try to start time at the same time, might have negative effect on server load. 

Instead, I thought that would be good to consider date with specific time, this way the likelihood of having colliding jobs will be lower than before, 

`2020-06-20T12:41:45.14Z`  when  time format is like this. 

I hope, I managed to explain the reason why I preferred to use `time.RFC3339`, yeah true, we might use it in any time parsing operation. 

@gianmarcomennecozzi